.catalog-onboard-pipeline.yaml

```diff
@@ -1,49 +1,15 @@
 ---
 apiVersion: v1
 offerings:
-  - name: deploy-arch-ibm-slz-vsi
+  - name: Wazi as a Service on VPC Landing zone
```

Comment:
This needs to be the programmatic name defined in your ibm_catalog.json
From looking at https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/pull/150 it seems to be `deploy-arch-ibm-wazi`